tests/integration/test_http_get_all.py

```diff
@@ -0,0 +1,93 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import utils
+
+
+def test_http_get_all(test_env, default_request, tmpdir):
+    """
+    Check HTTP GET_ALL works as expected.
```

Comment:
What do you think about this? `GET_ALL` has no real meaning so this makes it clear.
```suggestion
    Check the API endpoint to get all requests.
```